org.eclipse.scout.rt.ui.html/src/main/java/org/eclipse/scout/rt/ui/html/json/MultipartConfigFilter.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2010, 2024 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.scout.rt.ui.html.json;
```

Comment:
Can this be right? Wanted to place it near `UploadRequestHandler`

Reply:
Works for me. Near UploadRequestHandler is correct, the only caller of this part of handler are the ScoutJS widgets using Scout json protocol -> package ui.,html.json matches.

Reply:
Moved to org.eclipse.scout.rt.ui.html.app where we have access to both constants (Jetty and `UploadRequestHandler`).